docs/src/internals.md

```diff
@@ -20,18 +20,20 @@ A formula goes through a number of stages, starting as an
 expression that's passed to the `@formula` macro and ending up generating a
 numeric matrix when ultimately combined with a tabular data source:
 
-1. "Syntax time" when only the surface syntax is available
+1. "Syntax time" when only the surface syntax is available, during the defintion of `@formula`
```

Comment:
```suggestion
1. "Syntax time" when only the surface syntax is available, during the definition of `@formula`
```
But is "definition" the right word?

Reply:
yes I think definition is the rigth word
I am defining an `foo = @formula(y ~ x) `

Reply:
Precisely, I would say this is *calling* `@formula`. Defining it would be `macro formula...`.

Reply:
Right, we are defining _the formula `foo`_ by calling the `@formula` macro.
Or to put it another way:
_The definition of the formula `foo` is written in the DSL that is processed by the `@formula` macro_

I am not sure on good compact wording.

Reply:
I'd say "during the construction of the formula".

Reply:
```suggestion
1. "Syntax time" when only the surface syntax is available, when the `@formula` macro is invoked.
```

Reply:
@nalimilan is "invoking `@formula`" okay with you?